### PR TITLE
Create octofet package and add usage examples.

### PR DESCRIPTION
**Pull request by AlexErAmp**

### CODE REVIEW

File: examples/multiple_octofets.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+""" The program turns on all the switches in course, then turns them off."""
```
Comment (nkrkv) [inline examples/multiple_octofets.py]:
```suggestion
"""The program turns on all the channels one by one, then turns all off."""
```

File: examples/multiple_octofets.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import time
+import octofet
+
+
+def main():
+    print("Please enter the number of devices used.")
```
Comment (nkrkv) [inline examples/multiple_octofets.py]:
```suggestion
    print("Enter number of Octofets in chain (1 for single one):")
```

File: examples/octofet_control.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+"""Command line utility that allows you to control the state of switches on the
+Octofet board."""
```
Comment (nkrkv) [inline examples/octofet_control.py]:
```suggestion
"""Command line utility that allows you to control the state of switches on an
Octofet board."""
```

File: examples/octofet_control.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def main():
+    # Chip Enable pin. Set it to 0 or 1, depending on the connections.
+    pin_CE = 0
+    # Create an object for working with the Octofet board.
+    octo = octofet.Octofet(pin_CE)
+    try:
+        while True:
+            print("Enter the number of the switch you want to turn on/off.")
```
Comment (nkrkv) [inline examples/octofet_control.py]:
```suggestion
            print("Enter # of channel to manage (0 to 7):")
```

File: examples/octofet_control.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+            # Selected switch.
+            select_switch = int(input())
+            if select_switch < 0 or select_switch > 7:
+                print("The selected switch is not on the board, try again!")
```
Comment (nkrkv) [inline examples/octofet_control.py]:
```suggestion
                print("Error: Value should be in 0..7 range")
```

File: examples/octofet_control.py [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+                continue
+            print('Enter "on" to turn on the switch or "off" to turn it off.')
```
Comment (nkrkv) [inline examples/octofet_control.py]:
```suggestion
            print('Enter "on" to turn on the switch or "off" to turn it off:')
```

File: examples/octofet_control.py [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+            # The state in which the switch must be turned.
+            state = input()
+            if state == "on":
+                octo.digital_write(select_switch, True)
+            elif state == "off":
+                octo.digital_write(select_switch, False)
+            else:
+                print("Incorrect command, try again!")
```
Comment (nkrkv) [inline examples/octofet_control.py]:
```suggestion
                print("Error: Incorrect command, expected `on` or `off`")
```

**Review by nkrkv — CHANGES_REQUESTED** [review]

**Review by nkrkv — APPROVED** [review]
LGTM :ok_hand:

**Review by nomad605dis — APPROVED** [review]
😍

**Pull request merged**